README.md

```diff
@@ -1,22 +1,59 @@
 # MultiplayerSample Project
 
-A simple third-person multiplayer sample for O3DE.
+A third-person multiplayer game sample for O3DE. 
 
 > **_NOTE:_** For Linux, see the Linux specific setup in [README_LINUX](./README_LINUX.md).
 
+## Gameplay Overview
+This game sample has players competing to hit the highest score to win. Played over a series of rounds, each round brings greater opportunity for rewards but at higher risks.  Players race to collect gems scattered all over the level.
+
+Additionally, players have energy shields to protect them. Shooting other player damages their shields and getting hit damages your shields. Once your shields are depleted, you will respawn but at the cost of some of your collected gems.
+
+Do you risk it all to win? 
+
+Game features:
+* 3rd-person character setup.
+* Shooting with laser pistols.
+* A configurable number of rounds; defaults to 3.
+* Configurable gem spawning patterns per round to drive exploration.
+* Support for 1 to 15 players (goal is to raise performance to support 25 players per level).
+* Rich sounds and VFX support.
+* Teleporters to aid exploration.
+* Many points of extensibility.
+
+Note: There's no penalty for respawn if you are knocked off the level or use the teleporters.
+
+### Player Controls
+
+* Move using: `w,a,s,d` 
+* Sprint: Hold `shift`
+* Jump: `space` 
+* Look around: use mouse
+* Fire primary: `left mouse button`
+* See scoreboard: Hold `tab`
+* Open game menu: `esc`
+* Draw/holster active weapon: `e`
+
+To be implemented:
+* Fire secondary weapon: `right mouse button`
```

Comment:
This should be fire primary weapon

Reply:
I believe left mouse button is the primary weapon. Right mouse button isn't bound to anything and it would be secondary. So this is correct.

Reply:
I think @mcphedar just marked the wrong line, I think he's saying that the "Fire primary" line should say "Fire primary weapon"

Reply:
Or, never mind, just read his other comment. I'm wrong, though I think that would still be a good change...